packages/shorebird_cli/lib/src/xcodebuild.dart

```diff
@@ -72,7 +72,7 @@ class XcodeBuild {
       throw ProcessException(executable, arguments, '${result.stderr}');
     }
 
-    final lines = '${result.stdout}'.split('\n');
+    final lines = '${result.stdout}'.split('\n').map((e) => e.trim());
```

Comment:
minor but we could use `LineSplitter.split` here as well

Reply:
Neat! Done.